src/Main.purs

```diff
@@ -174,7 +174,7 @@ main = void do
   , jsonErrors
   } <- parseOptions (defaultOptions { cwd = cwd }) argv
 
-  let opts' = opts { libDirs = (_ <> Path.sep) <<< Path.resolve [cwd] <$> opts.libDirs }
+  let opts' = opts { libDirs = Str.drop (Str.length cwd + Str.length Path.sep) <<< (_ <> Path.sep) <<< Path.resolve [cwd] <$> opts.libDirs }
```

Comment:
I don't think this is quite right. This always assumes the lib directory is prefixed with cwd which isn't necessarily true. So perhaps we should just strip a `./` prefix if it exists?

Reply:
Hmm, given the possible permutations, maybe just not touching the provided arguments would be better then?